start-from-zero/index.css

```diff
@@ -246,21 +350,208 @@ textarea {
 
 #select {
     float: right;
+    -webkit-appearance: none; 
+    -moz-appearance: none;
+    appearance: none;   
+    background-image: url(assets/down-arrow.png);
+    background-repeat: no-repeat;
+    background-position: 90% 50%;
+    background-size: 13px;
+    position: relative;
+    height: 20px;
+    
+}
+
+
+
+textarea {
+    width: 99.7%;
+    margin: 0 auto;
+    padding: 0;
+}
+
+
+#footer-new {
+    position: relative;
 }
 
 fieldset {
+    margin: 30px 0;
+}
+
+.fieldset-wrap {
+    padding: 10px 10px;
+}
+
+fieldset legend {
+    padding: 0 20px;
+}
+
+#inner-li {
+    height: 100%;
+}
+
+.ultimate-wrap {
```

Comment:
I just wanted to wrap all my fieldsets in a div. I already had a fieldset-wrap class and I just named that class with the first thing that came into my mind. :P Thanks for pointing it out. That class was pretty useless anyway.